Tiamat.py

```diff
@@ -124,3 +134,9 @@ def print_msg(self, speaker, msg):
 
         self.msgfeed.config(state="disabled")
         self.msgfeed.see(tk.END)
+
+if __name__ == "__main__":
+    window = tk.Tk()
+    window.title("Tiamat")
+
+    Tiamat(window, run_in_idle=False)
```

Comment:
The file doesn't end with a new line character, which PEP 8 advocates for. In the future, please format all contributions with a formatter, e.g. Black.